README.md

```diff
@@ -0,0 +1,25 @@
+# ethereum-cache-creator
+## How to use program in the current iteration
+* download the [artifact](https://github.com/internet-sicherheit/ethereum-cache-creator/actions/runs/104782193)
+* unpack zip with a extractor of your choice, I would recommend [7zip](https://www.7-zip.de/).
+* open folder/directory in terminal
```

Comment:
```suggestion
* Open the extracted directory in a terminal
```

Reply:
The artifact itself does not contain a directory. when you press "extract here" there is no directory just the jar file.

Reply:
You are right.

So make it
`Open the directory containing the extracted JAR file in a terminal`